compiler/x/codegen/X86Ops.ins

```diff
@@ -2066,6 +2074,14 @@ INSTRUCTION(CMOVG8RegReg, cmovg,
             BINARY(VEX_L___, VEX_vNONE, PREFIX___, REX_W, ESCAPE_0F__, 0x4f, 0, ModRM_RM__, Immediate_0),
             PROPERTY0(IA32OpProp_ModifiesTarget | IA32OpProp_SourceRegisterInModRM | IA32OpProp_TestsOverflowFlag | IA32OpProp_TestsSignFlag | IA32OpProp_TestsZeroFlag),
             PROPERTY1(IA32OpProp1_LongSource | IA32OpProp1_LongTarget)),
+INSTRUCTION(CMOVGE4RegReg, cmovge,
+            BINARY(VEX_L___, VEX_vNONE, PREFIX___, REX__, ESCAPE_0F__, 0x4d, 0, ModRM_RM__, Immediate_0),
+            PROPERTY0(IA32OpProp_ModifiesTarget | IA32OpProp_SourceRegisterInModRM | IA32OpProp_IntSource | IA32OpProp_IntTarget | IA32OpProp_TestsOverflowFlag | IA32OpProp_TestsSignFlag | IA32OpProp_TestsOverflowFlag),
```

Comment:
`IA32OpProp_TestsOverflowFlag` is listed twice.

Reply:
Updated.